### ISSUE TITLE
[bug] cannot market sell

### ISSUE THREAD

**Issue by R-K-H** [Dodecahedr0x/meta-dao-frontend#85]
I think because 0 isn't being accepted as a price within the program. Could we use Number.MIN_VALUE or something similar?

**Comment by R-K-H** [comment]
In fact the only accepted value I can pass is `10` for the transaction to even accept and remove a base token from the wallet (in simulation).

**Comment by R-K-H** [comment]
Just FYI the market order for buy side works fine (or is working as expected). Is it because we truly don't need to pass in anything besides a correct value for `maxQuoteLotsIncludingFees`

If I set this up for the sell side of a market order 

```javascript
if (!limitOrder && ask) {
        priceLots = new BN(1);
        maxQuoteLotsIncludingFees = new BN(Math.floor(10 / QUOTE_LOTS));
      }
```
It accepts and works with any amount value I send it. Anything below `10/ QUOTE_LOTS` fails.

**Comment by R-K-H** [comment]
This is tentatively closeable, not sure on the one pending ^^^ ? above.

**Comment by metaproph3t** [comment]
> It accepts and works with any amount value I send it. Anything below `10/ QUOTE_LOTS` fails.

Are you able to submit the transaction to see what failure it comes up with? The logic looks right to me, although I don't know why you're getting this error

**Comment by R-K-H** [comment]
I guess my question is does `maxQuoteLotsIncludingFees` need to be related to the amount * price? or can it just be the minimal accepted value by the struct? On the market buy side it uses `priceLots.mul(quoteLots)` just didn't know if there was some acceptable value for "marketOrder" type.

**Comment by metaproph3t** [comment]
I don't think it does, it's just if `maxQuoteLotsIncludingFees` < amount*price that `maxQuoteLotsIncludingFees` matters
